bundles/org.openhab.binding.velbus/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -653,4 +969,401 @@
 			</options>
 		</event>
 	</channel-type>
+	<channel-type id="thermostat-trigger-channel">
+		<kind>trigger</kind>
+		<label>Thermostat Trigger Channel</label>
+		<event>
+			<options>
+				<option value="PRESSED">pressed</option>
+				<option value="RELEASED">released</option>
+			</options>
+		</event>
+	</channel-type>
+	<channel-group-type id="1channelInputModuleWithTemperatureSensor">
+		<label>Input Module with Temperature sensor</label>
```

Comment:
```suggestion
		<label>Input with Temperature Sensor</label>
```
The others don't have module in the label either.

Reply:
Modified in commit 5ca761c